app/controllers/memberships_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class MembershipsController < ApplicationController
+  before_action :authenticate!
+
+  # GET /memberships
+  # Endpoint functionality used by Users and Organizations
+  def index
+    membership = Membership.all
```

Comment:
Add check for parent param `organization_id` to display memberships for an organization with `Organization.find_by(params[:organization_id].memberships`.